src/modules/tasks.js

```diff
@@ -0,0 +1,211 @@
+        <p>Status: ${modalStatus(task.finished)} </p>
+        <p>Deadline: ${task.deadline} </p>
+      </div>
+    </div>
+  </div>
+</div>`;
+  });
+};
+
+const makeTaskSecOnClck = (value, selected) => {
+  if (!countProject) {
+    const tasksTitleDiv = document.createElement('div');
+    tasksTitleDiv.className = 'p-3';
+    tasksTitleDiv.innerHTML = `
+  <h2 class='prjct text-center mt-5'>Project: ${value}</h2>
+  <div class=" d-flex justify-content-center plus">
+    <h4 class="taskHeadline mt-1 me-5 mt-3 text-center">Tasks</h4>
+    <button type="button" class="pb-2 text-dark btn btn-light plus">
+      <i class="fa fa-plus plus"></i>
+    </button>
+  </div>`;
+
+    first.appendChild(tasksTitleDiv);
+    tasksToDom(selected);
+    countProject = true;
+  } else {
+    first.innerHTML = '';
+    tableArticle.innerHTML = '';
+    countProject = false;
+  }
+};
+
+const makeTaskSection = (value, selected) => {
+  first.innerHTML = '';
+  taskForm.innerHTML = '';
+  tableArticle.innerHTML = '';
+  const tasksTitleDiv = document.createElement('div');
+  tasksTitleDiv.className = 'p-3';
+  tasksTitleDiv.innerHTML = `
+  <h2 class='prjct text-center'>Project: ${value}</h2>
+  <div class=" d-flex justify-content-center plus">
+    <h4 class="taskHeadline mt-1 me-5 mt-3 text-center">Tasks</h4>
+    <button type="button" class="pb-2 text-dark btn btn-light plus">
+      <i class="fa fa-plus plus"></i>
+    </button>
+  </div>`;
+
+  first.appendChild(tasksTitleDiv);
+  tasksToDom(selected);
+  countProject = true;
+};
+
+const makeTaskForm = () => {
+  // const taskForm = document.createElement('form');
+  // taskForm.className = "taskForm border border-dark px-2 mx-5 ";
+  // taskForm.setAttribute('id', 'taskForm');
+  if (!countForm) {
+    taskForm.innerHTML = `
+    <input type='text' class=' my-2' name='taskName' id='task-name' placeholder='Add title here'>
+    <textarea  id='comment' class='my-2' rows='3' maxlength='150' placeholder='Add comment here'></textarea>
+    <select class='form-select me-4' id='selectPriority' aria-label='Default select example'>
+            <option selected value='1'>Select priority level</option>
+            <option class='one' value='1'>&#xf024; Priority 1</option>
+            <option class='two' value='2'>&#xf024; Priority 2</option>
+            <option class='three' value='3'>&#xf024; Priority 3</option>
+            <option class='four' value='4'>&#xf024; Priority 4</option>
+    </select>
+    <input type="date" max='2110-13-13' id='datePicker'>
+    <button type='button' class='confirmTask btn btn-secondary' id='confirmTask'>Create task</button>
+    <button type='button' class='dismissTask btn btn-light m-3' id='dismissTask'>Dismiss</button>
+  `;
+
+    document.getElementById('datePicker').setAttribute('min', setDate());
+    countForm = true;
+  } else {
+    taskForm.innerHTML = '';
+    countForm = false;
+  }
+};
+
+const changeStatus = (index) => {
+  // eslint-disable-next-line
```

Comment:
Kindly avoid the ESLInt inline disabling rule, it is the same as altering the ESLint config files